scripts/ci/is-community-pr.js

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env node
+
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/* eslint-env node */
+
+'use strict';
+
+const minimist = require( 'minimist' );
+
+const {
+	// The number of the associated GitHub or Bitbucket pull request. Only available on forked PRs.
+	CIRCLE_PR_NUMBER
+} = process.env;
+
+module.exports = main();
+
+function main() {
```

Comment:
I feel like this function is a bit confusing.
What about just exporting a const: `IS_COMMUNITY_PR`?
Then in the node command execution we could just use: 
![image](https://github.com/ckeditor/ckeditor5/assets/114912428/664c0f2d-20b4-4431-a9b3-2ca800af2c05)

Reply:
It's a compromise between having a single file that can be executed as CJS (`require()`) and a node script.

I am not glad of it, but it resolves issues with complicated bash ifs if we want to break the build.

Reply:
What do you think about splitting it into two files? The const could live in some `constants` directory/file. And the script could just use the const.

Reply:
That was the goal: having single file for this micro util :D

I will not block this change and I will update the scripts soon.